src/components/toolbox/copyToClipboard/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { CopyToClipboard as ReactCopyToClipboard } from 'react-copy-to-clipboard';
+import { translate } from 'react-i18next';
+import Icon from '../icon';
+import styles from './copyToClipboard.css';
+import Piwik from '../../../utils/piwik';
+
+class CopyToClipboard extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      copied: false,
+    };
+  }
+
+  textIsCopied() {
+    Piwik.trackingEvent('CopyToClipboard', 'button', 'Copy');
+    this.setState({
+      copied: true,
+    });
+    setTimeout(() => {
+      this.setState({
+        copied: false,
+      });
+    }, 3000);
+  }
+
+  render() {
+    const {
+      value, t, className, text, copyClassName,
+    } = this.props;
+    return (
+      <div onClick={(e) => {
+        e.stopPropagation();
+      }}>
+        <ReactCopyToClipboard text={value} onCopy={() => this.textIsCopied()}>
+          {this.state.copied ? <span className={`${className} copied`}>
+            {t('Copied!')}
+          </span> :
+            <span className={`${className} ${styles.clickable} default`}>
+              <span className='copy-title'>{text || value} </span>
+              <Icon name='copy' className={copyClassName}/>
+            </span>
+          }
+        </ReactCopyToClipboard>
+      </div>
+    );
+  }
+}
```

Comment:
IMO it would be good to set the default values for props `className` and `copyClassName` so if it's not passed it doesn't render `undefined` on the DOM